draft-ietf-quic-http.md

```diff
@@ -1414,8 +1414,8 @@ HTTP_STREAM_CREATION_ERROR (0x103):
 HTTP_CLOSED_CRITICAL_STREAM (0x104):
 : A stream required by the connection was closed or reset.
 
-HTTP_UNEXPECTED_FRAME (0x105):
-: A frame was received which was not permitted in the current state.
+HTTP_FRAME_UNEXPECTED (0x105):
+: A frame was received which was not permitted in the current state or stream.
```

Comment:
```suggestion
: A frame was received which was not permitted in the current state or on the current stream.
```
We've usually used "on" rather than "in" for a stream. (Though this reminds me of a great story about the choice of prepositions my former manager told once....)